clerk/tokens.go

```diff
@@ -69,18 +71,17 @@ func (c *client) VerifyToken(token string) (*Claims, error) {
 		return nil, fmt.Errorf("invalid signing algorithm %s", jwk.Algorithm)
 	}
 
-	claims := Claims{}
+	claims := SessionClaims{}
 	if err = parsedToken.Claims(jwk.Key, &claims); err != nil {
 		return nil, err
 	}
 
-	expectedClaims := jwt.Expected{
-		Audience: jwt.Audience{"clerk"},
-		Time:     time.Now(),
+	if err = claims.Claims.ValidateWithLeeway(jwt.Expected{Time: time.Now()}, 0); err != nil {
+		return nil, err
 	}
 
-	if err = claims.Claims.ValidateWithLeeway(expectedClaims, 0); err != nil {
-		return nil, err
+	if !strings.HasPrefix(claims.Issuer, "clerk.") {
```

Comment:
🔧 Shouldn't this be

```suggestion
	if !strings.HasPrefix(claims.Issuer, "https://clerk.") {
```

Reply:
Yes thats correct, i changed it in the middleware PR #23, although i will fix it here as it's the proper place